src/components/QuizDetail.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../stylesheet/Quiz.css';
```

Comment:
Do we need to import this stylesheet in every single file we are using classnames from it?

Reply:
Done